test/sharness/t0130-multinode.sh

```diff
@@ -53,6 +53,18 @@ check_file_fetch() {
 	'
 }
 
+test_resolve() {
+	_nid="$1"; shift
```

Comment:
When one want to use "$@", it's ok to use `shift`, so that the first arguments are not part of "$@". Here I think it would be better to use `$1`, `$2` and `$3` as there are other functions in the same file using `$1`, `$2` and `$3` already.